scss/_variables.scss

```diff
@@ -1 +1,3 @@
 $medium-text: 20px;
+$small-title: 16px;
+$small-text: 14px;
```

Comment:
Awesome!

I'd rename this filo into `font-sizes.scss` and use big, medium and small text. It's more generic than using title as a variable name